src/Framework/AspNetCoreAnalyzers/src/Analyzers/Resources.resx

```diff
@@ -321,4 +321,10 @@
   <data name="Analyzer_OverriddenAuthorizeAttribute_Title" xml:space="preserve">
     <value>[Authorize] overridden by [AllowAnonymous] from farther away</value>
   </data>
+  <data name="Analyzer_PublicPartialProgramClass_Message" xml:space="preserve">
+    <value>Using public partial class Program { } to make generated Program class public is no longer required. See https://aka.ms/aspnetcore-warnings/ASP0027 for more details.</value>
```

Comment:
```suggestion
    <value>Using public partial class Program { } to make generated Program class public is no longer required for ASP.NET Core projects. See https://aka.ms/aspnetcore-warnings/ASP0027 for more details.</value>
```
? Otherwise it may have the impression that C# takes care of it.

Reply:
Good note! Do you mind posting this on the API review for the analyzer over at https://github.com/dotnet/aspnetcore/issues/58488?